docs/contributing/publishing.md

```diff
@@ -1,14 +1,26 @@
 # Publishing
 
-This plugin uses Travis CI for publishing builds, so the following instructions are included only for reference purposes for the future if the build process requires change.
-
-* Create a personal access token
-  * Login to [JetBrains Hub](https://hub.jetbrains.com)
-  * Go to your user profile
-  * Click on the **Authentication** tab
-  * Click **New token ...**
-  * Enter any name
-  * Select **Plugin Repository** for the scope
-* Copy `gradle.properties.sample` to `gradle.properties` and enter your JetBrains Hub username and the token from the previous step.
-* Run `./gradlew publishPlugin` \(or Publish in IntelliJ\)
+This plugin uses GitHub Actions for publishing builds. The following instructions are a reference for maintainers.
 
+## Publishing a Release
+
+Follow the steps below to publish a new release. Once the GitHub release has been created, a GitHub action will be triggered which will build the plugin and push it to the plugin repository.
+
+1. Update the CHANGELOG.md file with the new version number and the change notes.
+1. Push all changes to master.
+1. Go to the [release page](https://github.com/one-dark/jetbrains-one-dark-theme/releases).
+1. Click the "Draft a new release" button.
+1. Enter the new version number for the tag and release name (e.g. `v1.0.0`).
+1. Add the change notes from the changelog to the description.
+1. Click "Publish release".
+
+## Creating a Personal Access Token
```

Comment:
I may be mistaken, but only your personal access token will be able to upload to the plugin repository, right? (I wouldn't want some random person to overwrite my plugin :)

Reply:
If we decide to keep this, do you think just linking to this might be better?

https://plugins.jetbrains.com/docs/marketplace/plugin-upload.html

Reply:
Correct.  These docs are just here for future reference.  I'm going to leave them as is as the link you sent isn't quite as specific to creating the token.